static/specification.html

```diff
@@ -93,34 +106,208 @@
+          using different [=payment end-point=].
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;video src="myvideo" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/video/pay"&gt;
+            &lt;/video&gt;
+
+            &lt;audio src="music.mp3" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/audio/pay"&gt;
+            &lt;/audio&gt;
+
+            &lt;picture srcset="cat.jpeg" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/images/pay"&gt;
+            &lt;/picture&gt;
+          </pre>
+        </aside>
+      </section>
+    </section>
+    <section id="model">
       <h2>
-        Terminology
+        Model
       </h2>
       <dl>
         <dt>
-          <dfn data-lt="provider">Web Monetization provider</dfn>:
+          <dfn data-lt="monetized">Monetization</dfn>:
+        </dt>
+        <dd>
+          A payment made by a user to a website, facilitated through a
+          [=monetization provider=].
+        </dd>
+        <dt>
+          <dfn data-lt="provider">Monetization provider</dfn>:
         </dt>
         <dd>
-          The party making payments on behalf of the user. A Web Monetization
-          provider uses the [[[SPSP]]] (<abbr title=
-          "Simple Payment Setup Protocol">SPSP</abbr>) on top of the
-          [[[Interledger]]] (<abbr title=
-          "Interledger Protocol">ILP</abbr>)/[[[STREAM]]] to provide a
-          high-level way to send money and data.
+          The party making payments on behalf of the user. A monetization
+          provider leverages the [[[Interledger]]] suite of protocols and
+          technologies (e.g., [[Open-Payments]] or [[STREAM]]) to provide a
+          high-level way to pay a [=monetization receiver=].
         </dd>
         <dt>
-          <dfn>Web Monetization receiver</dfn>:
+          <dfn>Monetization receiver</dfn>:
         </dt>
         <dd>
-          The party receiving payments on behalf of the website.
+          The party receiving payments on behalf of the website, whose details
+          are provided by a [=payment end-point=].
         </dd>
         <dt>
           <dfn>Payment end-point</dfn>:
         </dt>
         <dd>
-          A [=URL=] to a conforming HTTP <a data-cite="SPSP#specification">SPSP
-          end-point</a>.
+          A [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
+          (i.e., a JSON resource containing details that facilitate payment).
+        </dd>
+        <dt>
+          <dfn>Payment connection</dfn>
+        </dt>
+        <dd>
+          A network connection made over the [[[Interledger]]] network using,
+          for example, [[Open-Payments]] or [[STREAM]], for the purpose of
+          facilitating a payment.
```

Comment:
same as above

However, I feel like it should not just be 
> A network connection made over the Interledger Protocol network using, for example, [Open-Payments] or [SPSP], both based on [STREAM], for the purpose of facilitating a payment.

I'm hesitating because 
* when using SPSP, a STREAM connection is actually opened by the agent
* when using OP, the agent will create an outgoing payment at a wallet, which then will open the STREAM connection

Maybe "payment connection" is not the right term. I'll think about that a bit more. 

Maybe @adrianhopebailie has an idea how to rephrase that to make it clearer.

Reply:
The point I wanted to make here was, from the point of view of a browser: "Web Monetization over HTTP (OP)" vs "Web Monetization over STREAM". The Web Monetization Agent in the browser decides which transport layer to use to make that connection in order to start payments. "Payment connection" was the best description we could come up with, not the ideal term. Looking forward to a better term.

Reply:
Ok, happy to change this... everyone, give this a think and let's see if we can come up with something better for when we meet next week! ⚗️

Reply:
Oh! I REALLY like the precision, @sabinebertram! Just to unpack it, I split the conditionals into an unordered list. 

What do you think?

Reply:
Perfect.